Update Porting eBook (#27512)
dotnet/docs: docs/architecture/blazor-for-web-forms-developers/components.md

<MyComponentLib.Counter />
```

> ## Modifying page title from components

Reviewer: ```suggestion
## Modify page title from components
```

<Counter IncrementAmount="10" />
```

> ### Querystring parameters

Reviewer: ```suggestion
### Query string parameters
```

> Blazor components can also leverage values from the querystring of the page they're rendered on as a parameter source. To enable this, add the `[SupplyParameterFromQuery]` attribute to the parameter. For example, the following parameter definition would get its value from the request in the form `?IncBy=2`:

Reviewer: ```suggestion
Blazor components can also leverage values from the query string of the page they're rendered on as a parameter source. To enable this, add the `[SupplyParameterFromQuery]` attribute to the parameter. For example, the following parameter definition would get its value from the request in the form `?IncBy=2`:
```

```csharp

>     [Parameter]
>     [SupplyParameterFromQuery(Name = "IncBy")]
>     public int IncrementAmount { get; set; } = 1;

Reviewer: ```suggestion
[Parameter]
[SupplyParameterFromQuery(Name = "IncBy")]
public int IncrementAmount { get; set; } = 1;
```

```

If you don't supply a custom `Name` in the `[SupplyParameterFromQuery]` attribute, by default it will match the property name (`IncrementAmount` in this case).

> ## Components and Error Boundaries

Reviewer: ```suggestion
## Components and error boundaries
```

By default, Blazor apps will detect unhandled exceptions and show an error message at the bottom of the page with no additional detail. To constrain the parts of the app that are impacted by an unhandled error, for instance to limit the impact to a single component, the `<ErrorBoundary>` tag can be wrapped around component declarations.

> For example, to protect against possible exceptions thrown from the Counter component, declare it within an `<ErrorBoundary>` and optionally specify a message to display if there is an exception:

Reviewer: ```suggestion
For example, to protect against possible exceptions thrown from the `Counter` component, declare it within an `<ErrorBoundary>` and optionally specify a message to display if there is an exception:
```
